tests/test_gestalt.py

```diff
@@ -7,6 +7,49 @@
 import hvac
 
 
+# Testing member function
+def test_combined_into():
```

Comment:
nit: name `test_combine_into` to match function name that it's testing

Reply:
I updated the name of the function merge_into. and updated the test name as well